container/libcontainer/handler.go

```diff
@@ -299,8 +299,9 @@ func processStatsFromProcs(rootFs string, cgroupPath string, rootPid int) (info.
 		}
 	}
 
+	processCount := common.ReadUInt64(cgroupPath, "pids.current")
```

Comment:
Can you please add a specific method in helpers.go instead of exposing the method `readUInt64`?

Reply:
I exposed it because we already had this function elsewhere. I can copy it and bring it here also.

Reply:
I realized I didn't need to export readString but I still need to export this.

I exported this from common so that I could use it in libcontainer.

Reply:
@dims wdyt?

I can copy duplicate ReadUInt64 into libcontainer if we don't want to export the function. It just means we will have the same function in two locations.